addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -194,6 +194,15 @@ def format_alphanumeric(text_to_convert):
             if rc_refund:
                 document_total = -abs(document_total)
 
+        # If the document follows a downpayment / downpayments, the
+        downpayment_dict = {}
+        for line in self.invoice_line_ids:
+            moves = line.search(
+                [('sale_line_ids', 'in', line.sale_line_ids.ids)]
```

Comment:
@dbkosky sale_line_ids does not exist here

Reply:
Was not the idea to change _is_downpayment to _get_downpayment and do this calculation there?